src/uwtools/tests/config/formats/test_fieldtable.py

```diff
@@ -10,10 +10,19 @@
 # Tests
 
 
-def test_format():
+def test_get_format():
     assert FieldTableConfig.get_format() == FORMAT.fieldtable
 
 
+def test_get_depth_threshold():
+    assert FieldTableConfig.get_depth_threshold() is None
+
+
+def test_instantiation_depth():
+    # Any depth is fine.
+    assert FieldTableConfig(config={1: {2: {3: 4}}})
```

Comment:
Now that I'm seeing this written out, I'm not sure if it's true. I was thinking we had a JSON Schema for this file type, but I was mistaken. At any rate, it is currently true, and we likely need to put some bounds on it in a subsequent PR.